enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -9,15 +9,38 @@ import { saveDomImageStyles } from "metabase/visualizations/lib/save-chart-image
  * even when rendered under a React portal.
  */
 export const PublicComponentStylesWrapper = styled.div`
+  // Try to reset as much as possible to avoid css leaking from host app to our components
+  all: initial;
+  text-decoration: none;
+
+  // # Basic css reset
+  // We can't apply a global css reset as it would leak into the host app
+  // but we can't also apply our entire css reset scoped to this container,
+  // as it would be of higher specificity than some of our styles.
+  // We'll have to hand pick the css resets that we neeed
+
+  button {
+    border: 0;
+    background-color: transparent;
+  }
+  // end of RESET
+
+  font-style: normal;
```

Comment:
the css properties added below this are the ones i found on `rootStyle`, that was previously added at the MetabaseProvider level.
I initially had placed the entire rootStyle here but most of the properties were duplicated or not needed